public/login.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<title>LetsVybe logIn Page</title>
+
+<head>
+  <link type="text/css" rel="stylesheet" href="styles/login.css" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+</head>
+
+<body>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <!-- Bootstrap CSS -->
+
+  <div class="bg-image">
+    <h1 class="title blinking">LETSVYBE</h1>
+  </div>
+
+  <div class="content">
+
+    <div class="title">Create account</div>
+    <input type="text" placeholder="E-mail" />
+    <input type="password" placeholder="Password" />
+    <input type="checkbox" id="rememberMe" />
```

Comment:
For some reason this doesn't show up well in the browser. Try fixing it.

Reply:
Yes I realized that is not working how it is supposed to! I will figure it out by tonight!